security-release-process-documentation/security-release-process.md

```diff
@@ -286,3 +318,318 @@ Yes.
 CrashOverride will vouch for Acidburn joining the list on behalf of the "Seven"
 distribution.
 ```
+
+## Severity Thresholds - How We Do Vulnerability Scoring
+
+The content presented below outlines basic criteria when considering the effect
+of bugs disclosed to the Product Security Team and their severity. These apply
+mostly to items in "core" but could be abstractly applied to those in other
+repos as well.
+
+### Server
+
+The "server" pertains to any "server-side components" (see below) we distribute,
+but you can specifically think of `kube-apiserver` and `kubelet` being the main ones.
+
+"Server-side components" pertains to:
+
+- Components built from source in the `kubernetes` github organization and 
+  [all current organizations in use](https://github.com/kubernetes/community/blob/master/org-owners-guide.md#current-organizations-in-use)
+- Container images under the `k8s.gcr.io` repository
+- Containers included in the [core addons directory](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons)
+
+The server bar is usually not appropriate when user interaction is part of 
```

Comment:
Having a hard time understanding "server bar", what does it mean in this context?

Reply:
reworded a bit